# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/HostWebGLContext.h
# Commit: adc841b3e204
# Full Hash: adc841b3e204564ee8515a87206eb317275844c4
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 09:51:10
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - SurfaceFromElement gets alpha Premult unless opt-in to NonPremult. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63421
# ==============================================================================

diff -r 7917b3a4a9a7 -r adc841b3e204 dom/canvas/HostWebGLContext.h
--- a/dom/canvas/HostWebGLContext.h	Fri Feb 21 03:11:23 2020 +0000
+++ b/dom/canvas/HostWebGLContext.h	Fri Feb 21 02:27:08 2020 +0000
@@ -196,11 +196,6 @@
 
   void DidRefresh() { mContext->DidRefresh(); }
 
-  RefPtr<gfx::SourceSurface> GetSurfaceSnapshot(
-      gfxAlphaType* out_alphaType) const {
-    return mContext->GetSurfaceSnapshot(out_alphaType);
-  }
-
   void GenerateError(const GLenum error, const std::string& text) const {
     mContext->GenerateErrorImpl(error, text);
   }